master/internal/kubernetes/request_workers.go

```diff
@@ -0,0 +1,97 @@
+package kubernetes
+
+import (
+	"github.com/determined-ai/determined/master/pkg/actor"
+
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	typedV1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type requestProcessingWorker struct {
+	podInterface       typedV1.PodInterface
+	configMapInterface typedV1.ConfigMapInterface
+}
+
+func (r *requestProcessingWorker) Receive(ctx *actor.Context) error {
+	switch msg := ctx.Message().(type) {
+	case actor.PreStart:
+		ctx.Tell(ctx.Self().Parent(), workerAvailable{})
+
+	case createKubernetesResources:
+		r.receiveCreateKubernetesResources(ctx, msg)
+
+	case deleteKubernetesResources:
+		r.receiveDeleteKubernetesResources(ctx, msg)
+
+	default:
+		ctx.Log().Errorf("unexpected message %T", msg)
+		return actor.ErrUnexpectedMessage(ctx)
+	}
+	return nil
+}
+
+func (r *requestProcessingWorker) receiveCreateKubernetesResources(
+	ctx *actor.Context,
+	msg createKubernetesResources,
+) {
+	defer ctx.Tell(ctx.Self().Parent(), workerAvailable{})
```

Comment:
nonblocking: if you could move this line to `Receive`, it would be more orderly IMO.